spec/PhpSpec/Formatter/Presenter/Value/BaseExceptionTypePresenterSpec.php

```diff
@@ -28,4 +28,17 @@ function it_should_present_an_error_as_a_string()
         $this->present(new ErrorException(new \Error('foo')))
             ->shouldReturn('[err:Error("foo")]');
     }
+
+    function it_should_present_a_parse_error_with_file_and_line_number()
+    {
+        $this->present(new ErrorException(new class() extends \ParseError {
+            public function __construct()
+            {
+                $this->message = 'Something is not correct';
+                $this->file = '/app/some/file.php';
+                $this->line = 42;
+            }
+        }))
+            ->shouldContain('Something is not correct in "/app/some/file.php" on line 42');
```

Comment:
Another suggestion would be `Something is not correct in /app/some/file.php:42`.

Some cool terminal might allow to quick open the file (by clicking on the path) in one's favorite editor. I've tried unsuccessfully with `guake` though => https://guake.readthedocs.io/en/stable/user/features.html#quick-open